site/en/docs/extensions/mv3/intro/index.md

```diff
@@ -1,51 +1,60 @@
 ---
 layout: 'layouts/doc-post.njk'
-
-# The page title. This appears at the top of the doc and as the page name
-# in Google Search.
 title: Welcome to Manifest V3
-
-# This appears below the title and is an optional teaser
 subhead: 'A step in the direction of security, privacy, and performance.'
-
-# This appears in the ToC of the project landing page at
-# /docs/[project-name]/. It also appears in the <meta description> used in 
-# Google Search.
 description: 'An introduction to Manifest V3'
-
-# The publish date
 date: 2020-11-09
+updated: 2022-09-02
+---
 
-# An optional updated date
-# updated: 2020-10-16
+This site introduces Manifest V3, short for Manifest Version 3, which is the latest iteration of the Chrome extension platform. It shares the background and reasons for introducing Manifest V3 and the vision for the platform's future, along with resources on how to migrate.
 
-# A list of authors. These usernames correspond to the keys in the
-# _data/authorsData.json file.
+## What is a Manifest?
 
----
+An extension [Manifest][doc-manifest] gives the browser information about the extension, such as
+the most important files and the capabilities the extension might use. The extension platform features change when there's a new [Manifest version][manifest-version].
+
+## Introducing Manifest V3 
+
+Manifest V3 represents one of the most significant shifts in the extensions platform since it
```

Comment:
```suggestion
Manifest V3 is one of the most significant shifts in the extensions platform since it
```

Reply:
See [definition 2](https://www.google.com/search?q=represents&oq=represents&aqs=chrome..69i57j0i131i433i512j69i65j69i61l3j69i65l2.1370j0j7&sourceid=chrome&ie=UTF-8). I'm not sure that 'amounts to' is what you had in mind here. If 'is' is too strong, the other option is to rewrite the sentence.

cc: @dotproto

Reply:
I'll merge when you all decided what to do with this.

Reply:
Amy is going to merge (or maybe already has merged) this change.

For context, the use of "represets" here was intentional. We were saying that not only is Manifest V3 itself a major change, it is also the manifestation of a shift in a philosophical shift in how we're thinking about browser extensions. Another major change coming is a change to how Chrome approaches host permissions grants; in the future the browser will default to run "when you click the extension" rather than "on all sites". There are also other manifestations, such as how we approach API design and the review process.

Ultimately, though, Manifest V3 itself is a huge change and the subtlety of "is" vs. "represents" in this context is mostly just debating the color of the bikeshed.